buildSrc/src/main/java/AndroidComponents.kt

```diff
@@ -3,5 +3,5 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 object AndroidComponents {
-    const val VERSION = "63.0.20201014190200"
+    const val VERSION = "63.0.20201015143124"
```

Comment:
Can we leave this update out of the PR and wait for the normal AC update?

Reply:
@eliserichards yes, this is just here so the PR compiles, as I need a new action introduced in this version. The A-C update will land before this PR anyway.